src/view/main-mobile.js

```diff
@@ -215,6 +215,7 @@ const PayStack = createStackNavigator(
   {
     Pay,
     PayLightningConfirm,
+    Wait,
```

Comment:
So confused why this fixes the bug :thinking:

Reply:
> So confused why this fixes #921 🤔

Correct. The navigator will now slide right from the after the user confirms the transaction. again the stack navigator got all mixed up since we had the wait screen wired up on the main stack navigator instead of the payment stack. The change includes the screen in both stacks for a more consistent transitions.